### DISCUSSION TITLE
Github Account Recovery

### DISCUSSION THREAD

**Discussion by ghost** [community/community#34373] [Other Feature Feedback, Questions, & Ideas]
Deat Community My name is Muhammad Shahrukh I have been using GitHub since 2017  above, I also set google authentication for my account and also save the recovery code I also face difficulties by crashing my OS and windows as well but now at this condition, My 2fact and recover didn't work on the same account which is also the single account. waiting for the response
&
I will be thankful if someone helps

**Comment by queenofcorgis** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
